owncloudData/src/test/java/com/owncloud/android/data/file/datasources/OCLocalFileDataSourceTest.kt

```diff
@@ -683,6 +683,28 @@ class OCLocalFileDataSourceTest {
 
     }
 
+    @Test
+    fun `updateFileWithLastUsage should change last usage parameter with current time for a file`() {
+
+        val currentTime = System.currentTimeMillis()
+
+        every { dao.updateFileWithLastUsage(OC_FILE.id!!, currentTime) } returns Unit
+
+        localDataSource.updateFileWithLastUsage(OC_FILE.id!!, currentTime)
+
+        verify(exactly = 1) { dao.updateFileWithLastUsage(OC_FILE.id!!, currentTime) }
+    }
+
+    @Test
+    fun `updateFileWithLastUsage should change last usage parameter with null for a file`() {
+
+        every { dao.updateFileWithLastUsage(any(), null) } returns Unit
```

Comment:
```suggestion
        every { dao.updateFileWithLastUsage(OC_FILE.id!!, null) } returns Unit
```
Why here `any()` and in the previous one `OC_FILE.id!!`? Let's be consistent

Reply:
Done